### PR TITLE
Update `pallet-multisig` benches

### PR DESCRIPTION
**Pull request by ggwpez**
The multisig weights were not updated in https://github.com/paritytech/substrate/pull/12072 and now the trait does not fit together with the benchmarks; possibly blocking the next release. Build error from https://github.com/paritytech/polkadot/pull/6173:  

```pre
error[E0046]: not all trait items implemented, missing: `as_multi_create_store`, `as_multi_approve_store`, `approve_as_multi_complete`
    --> /builds/parity/mirrors/polkadot/runtime/kusama/src/weights/pallet_multisig.rs:46:1
     |
  46 | impl<T: frame_system::Config> pallet_multisig::WeightInfo for WeightInfo<T> {
     | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ missing `as_multi_create_store`, `as_multi_approve_store`, `approve_as_multi_complete` in implementation
```
cc @ruseinov (PS: Sorry wrong @ here before) @coderobe 

Polkadot Companion: https://github.com/paritytech/polkadot/pull/6188  
Cumulus companion: https://github.com/paritytech/cumulus/pull/1805

### CODE REVIEW

**Comment by ggwpez** [comment]
/cmd queue -c bench-bot $ pallet dev pallet_multisig

**Review by bkchr — APPROVED** [review]

**Review by shawntabrizi — APPROVED** [review]

**Comment by shawntabrizi** [comment]
/cmd queue -c bench-bot $ pallet dev pallet_multisig

File: frame/multisig/src/lib.rs [hunk 1]
```diff
@@ -434,7 +433,6 @@ pub mod pallet {
 
 			T::WeightInfo::approve_as_multi_create(s)
 				.max(T::WeightInfo::approve_as_multi_approve(s))
-				.max(T::WeightInfo::approve_as_multi_complete(s))
```
Comment (ggwpez) [inline frame/multisig/src/lib.rs:437]: I am not sure if this is correct. @ruseinov do you remember why the `approve_as_multi_complete` benchmark was removed?  
Why do we not need it anymore?

Comment (ggwpez) [reply frame/multisig/src/lib.rs:437]: Okay seems fine since the bench does not make sense anymore.

**Review by coderobe — APPROVED** [review]

**Comment by ggwpez** [comment]
bot merge

**Pull request merged**